pyproject.toml

```diff
@@ -53,6 +53,7 @@ pytest-asyncio = ">= 0.12"
 pytest-cov = "*"
 types-docutils = "*"
 types-PyYAML = "*"
+types-aiofiles = "*"
```

Comment:
The main reason why we didn't do that yet (I think) is that I wasn't aware there are types for this. (Or maybe there weren't when I checked? No idea...)